### PR TITLE
The logging guide review for 3.15

### PR DESCRIPTION
**Pull request by MichalMaler**
Grammar and style review for the logging guide.

### CODE REVIEW

File: docs/src/main/asciidoc/logging.adoc [hunk 1]
```diff
@@ -217,7 +218,7 @@ quarkus.log.category."org.hibernate".level=DEBUG
 ----
 
 When you set the log level to below `DEBUG`, you must also adjust the minimum log level.
-This setting is either global, using the `quarkus.log.min-level` configuration property, or per category:
+You can apply this setting either globally with the `quarkus.log.min-level` configuration property, or per category:
```
Comment (dmlloyd) [inline docs/src/main/asciidoc/logging.adoc:221]: Are we doing second-person here? Maybe instead:
```suggestion
This setting may be applied either globally with the `quarkus.log.min-level` configuration property, or per category:
```

**Review by dmlloyd — APPROVED** [review]
LGTM, one minor comment, take it or leave it.

**Review by sheilamjones — COMMENTED** [review]
Hi @MichalMaler,
I completed peer review and please see just a small few comments for your consideration. One general comment is about the use of the  {project-name} attribute as I think this renders to "Red Hat build of Quarkus" and not "Quarkus" and if this is your intention.
Kind regards,
Sheila

**Review by Ladicek — APPROVED** [review]
LGTM, although using "Red Hat build of Quarkus" everywhere in the product rendering seems excessive.

File: docs/src/main/asciidoc/logging.adoc [hunk 2]
```diff
@@ -10,10 +10,10 @@ include::_attributes.adoc[]
 :diataxis-type: reference
 :topics: logging,observability
 
-Read about the use of logging API in Quarkus, configuring logging output, and using logging adapters to unify the output from other logging APIs.
+Read about the use of logging API in {project-name}, configuring logging output, and using logging adapters to unify the output from other logging APIs.
```
[the remark this answers is not in the document]
Comment (MichalMaler) [reply docs/src/main/asciidoc/logging.adoc:13]: @sheilamjones Hello Sheila. Here in Quarkus, it renders as `Quarkus`, while in RHBQ repo, it renders as `Red Hat build of Quarkus`.

**Comment by MichalMaler** [comment]
> LGTM, although using "Red Hat build of Quarkus" everywhere in the product rendering, seems excessive.

OK, I will keep it only in Level 1 and Level 2 heading sections and only in its first appearance.

File: docs/src/main/asciidoc/logging.adoc [hunk 3]
```diff
@@ -57,7 +57,7 @@ In such cases, you need to use a <<logging-apis,logging adapter>> to ensure that
 
 == Get an application logger
 
-In Quarkus, the most common ways to obtain an application logger are by:
+In {project-name}, the most common ways to obtain an application logger are by:
```
[the remark this answers is not in the document]
Comment (MichalMaler) [reply docs/src/main/asciidoc/logging.adoc:60]: OK, thank you!

File: docs/src/main/asciidoc/logging.adoc [hunk 4]
```diff
@@ -152,24 +152,25 @@ class SimpleBean {
...
-WARN:: A non-critical service error or problem that may not require immediate correction.
-INFO:: Service lifecycle events or important related very low-frequency information.
-DEBUG:: Messages that convey extra information regarding lifecycle or non-request-bound events, useful for debugging.
-TRACE:: Messages that convey extra per-request debugging information that may be very high frequency.
-ALL:: A special level to use in configuration to turn on logging for all messages, including custom levels.
+OFF:: A special level used in configuration to turn off logging.
+FATAL:: A critical service failure or total inability to handle any requests.
+ERROR:: A major issue in processing or an inability to complete a request.
+WARN:: A non-critical service error or problem that might not require immediate correction.
+INFO:: Service lifecycle events or other important information that occurs infrequently.
```
[the remark this answers is not in the document]
Comment (MichalMaler) [reply docs/src/main/asciidoc/logging.adoc:170]: I think that occurs is a frequently used term in software development, logging, and debugging. I am for keeping it but will ask a SME for an additional opinion  :)

File: docs/src/main/asciidoc/logging.adoc [hunk 5]
```diff
@@ -57,7 +57,7 @@ In such cases, you need to use a <<logging-apis,logging adapter>> to ensure that
 
 == Get an application logger
 
-In Quarkus, the most common ways to obtain an application logger are by:
+In {project-name}, the most common ways to obtain an application logger are by:
 
 * <<declaring-a-logger-field>>
 * <<simplified-logging>>
```
[the remark this answers is not in the document]
Comment (MichalMaler) [reply docs/src/main/asciidoc/logging.adoc:63]: Originally, there was Logging with Panache, which many people find confusing; thus, we agreed to rename it that way. Besides, it is a settled term :)

File: docs/src/main/asciidoc/logging.adoc (hunk 4 above)
[the remark this answers is not in the document]
Comment (Ladicek) [reply docs/src/main/asciidoc/logging.adoc:170]: I kinda agree that "information occurs" doesn't sound right. This

>  INFO:: Service lifecycle events or other important infrequent information.

sounds good to me.

[the remark this answers is not in the document]
Comment (MichalMaler) [reply docs/src/main/asciidoc/logging.adoc:170]: Great! Thank you both @Ladicek @sheilamjones . Applying! :)

File: docs/src/main/asciidoc/logging.adoc (hunk 1 above)
Comment (dmlloyd) [reply docs/src/main/asciidoc/logging.adoc]: No, in this case keeping `may` is appropriate I think; I would use `might` for something that could happen or is allowed to happen out of the user's control (but is not certain), but use `may` for something that the user is specifically being permitted or suggested to do.

**Comment by MichalMaler** [comment]
> Hi @MichalMaler, I completed peer review and please see just a small few comments for your consideration. One general comment is about the use of the {project-name} attribute as I think this renders to "Red Hat build of Quarkus" and not "Quarkus" and if this is your intention. Kind regards, Sheila

Yup, all should be Gucci :)

**Pull request merged**